config/sidekiq.yml

```diff
@@ -1,7 +1,6 @@
 ---
 :concurrency: 5
 :pidfile: tmp/sidekiq.pid
-:logfile: log/sidekiq.log
```

Comment:
I think we want to log sidekiq as well right?

Reply:
It does, but standard to STDOUT. This allows for proper logging to be picked up by other services in a standard containerized environment. We could allow configuring this via ENV vars.

Reply:
👍